<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DB_CommandWidget</class>
 <widget class="QWidget" name="DB_CommandWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Command Line</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTextEdit" name="teHistory">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lCommand">
       <property name="text">
        <string>Command:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DB_CommandEdit" name="leCommand" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DB_CommandEdit</class>
   <extends></extends>
   <header>db_commandedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
